.github/workflows/release_by_tag.yml

```
@@ -0,0 +1,66 @@
+
+on: 
+  push:
+    tags: [ "v*" ]
+
+permissions: write-all
```

This is a potentially a security hazard, as it grants the workflow write access to everything.

From what I can see, only the `contents` permission needs to be `write`.
Could you restrict the permissions to only what is required:
```suggestion
permissions:
  contents: write
```